web-client/src/pages/routes/NewRequestRoute/NewRequestRoute.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactElement } from 'react';
```

Comment:
Maybe change the name of this file to NewRequestFormRoute.
@jpmarques66 opinion?

Reply:
Here I wouldn't change it because this is in fact the "New Request" page and it can have more that just a form.